test/claims-status/components/appeals-v2/AppealsV2StatusPage.unit.spec.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import { mockData } from '../../../../src/js/claims-status/utils/helpers';
+import AppealsV2StatusPage from '../../../../src/js/claims-status/containers/AppealsV2StatusPage';
+
+describe.only('<AppealsV2StatusPage/>', () => {
+  const defaultProps = { appeal: mockData.data[0] };
+
+  it('should render', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    expect(wrapper.type()).to.equal('div');
+  });
+
+  it('should render an unexpanded <Timeline/>', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const timeline = wrapper.find('Timeline').dive();
+    const expander = timeline.find('Expander');
+    const pastEvents = timeline.find('PastEvent');
+    expect(expander.length).to.equal(1);
+    expect(pastEvents.length).to.equal(0);
+  });
+
+  it('should pass down events as props to Timeline', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const pastEvents = wrapper.find('Timeline').props().events;
+    expect(pastEvents.length).to.equal(defaultProps.appeal.attributes.events.length);
+  });
+
+  it('should render a <CurrentStatus/> with title and description', () => {
+    const wrapper = shallow(<AppealsV2StatusPage {...defaultProps}/>);
+    const statusProps = wrapper.find('CurrentStatus').props();
+    // Testing against a specific string would require duplicating component
+    // and util code so here we're just testing that both props exist
```

Comment:
Yup, I think this is great. These specific tests should be handled in the tests for CurrentStatus, methinks.